esileapclient/tests/functional/v1/test_basic.py

```diff
@@ -213,6 +214,36 @@ def test_lease_show_offer_deleted(self):
         details = esi.lease_show(self.clients['child-lessee'], lease['uuid'])
         self.assertEqual(details['status'], 'deleted')
 
+    def test_node_list_long(self):
+        """ Tests functionality of "esi node list" if using node_uuid.
+            returns the list of nodeif optional argument is not passed.
+            Test steps:
+            1) Set the environment variable using
+               export OS_FUNCTIONAL_NODE_IDENT_UUID = node_uuid
+            2) Check that node details were returned.
+            3) Check that the output of "node list" contains
+               the node uuid mentioned. """
+
+        node_uuid = (os.getenv('OS_FUNCTIONAL_NODE_IDENT_UUID'))
+        listings = esi.node_list(self.clients['admin'], long=True)
+        self.assertNotEqual(listings, [])
+        self.assertIn(node_uuid, [x['UUID'] for x in listings])
+
+    def test_node_list_basic(self):
+        """ Tests basic functionality of "esi node list" it by default
+            returns the list of node name if optional argument is not passed.
+            Test steps:
+            1) Set the environment variable using
+               export OS_FUNCTIONAL_NODE_IDENT_NAME = node_name
+            2) Check that node details were returned.
+            3) Check that the output of "node list" contains
+               the node name mentioned. """
+
+        node_name = (os.getenv('OS_FUNCTIONAL_NODE_IDENT_NAME'))
+        listings = esi.node_list(self.clients['admin'])
+        self.assertNotEqual(listings, [])
+        self.assertIn(node_name, [x['Name'] for x in listings])
```

Comment:
Pretty much everything you change in the previous test should also be changed in this test, except for the extra environment variable check.

Reply:
Done the same changes in both functions.